### PR TITLE
[DC-1520] Add primary_pid_rid_mapping checks to the clean rdr qc notebook

### PR DESCRIPTION
**Pull request by brendagutman**

### CODE REVIEW

File: data_steward/analytics/cdr_ops/clean_rdr_export_qc.py [hunk 1]
```diff
@@ -604,6 +604,60 @@
...
+#
+# The first check is for finding any duplicates that exist in primary_pid_rid_mapping table. If there are issues, `'duplicates_in_pprm'` will appear in the output dataframe along with the count of this occurance. This check failing implies the CR did not run as expected and created the duplicates in primary_pid_rid_mapping.
+#
+# The second check is for finding any records in the rdr export pid_rid_mapping table that don't exist in the primary_pid_rid_mapping table. If there are issues, `'prm_missing_from_pprm'` will appear in the output dataframe along with the count of this occurance. This check failing implies the CR did not run as expected. All new records should have been appended to primary_pid_rid_mapping.
+#
+# The third check ensures the shift column contains numbers between 1 and 135(exclusive). If there are issues, `'shift_off_spec'` will appear in the output dataframe along with the count of this occurance. This check failing implies the CR did not run as expected. All records in pprm should have a shift between 1 and 365.
```
Comment (lrwb-aou) [inline data_steward/analytics/cdr_ops/clean_rdr_export_qc.py:619]: Suggestion:
```suggestion
# The third check ensures the shift column contains numbers between 1 and 365(exclusive). If there are issues, `'shift_off_spec'` will appear in the output dataframe along with the count of this occurance. This check failing implies the CR did not run as expected. All records in pprm should have a shift between 1 and 365.
```

**Review by lrwb-aou — APPROVED** [review]
@brendagutman Nice PR!  Just let me know if 135 wasn't a typo.  I think it was.

**Comment by brendagutman** [comment]
> @brendagutman Nice PR! Just let me know if 135 wasn't a typo. I think it was.

You are correct @lrwb-aou !  I made the update and will merge this when the checks pass. Thanks for finding that!

**Comment by brendagutman** [comment]
replaced by another PR. Branch bg/dc_1520_a

**Pull request closed without merging**